src/config/mod.rs

```diff
@@ -135,6 +139,37 @@ impl Config {
         Ok(cfg)
     }
 
+    /// Build out a config for the workspace root, which is only interested in what's in the
+    /// oranda.json.
+    pub fn build_workspace_root(config_path: &Utf8PathBuf) -> Result<Config> {
+        let conf = OrandaLayer::load(config_path)?;
+        let mut cfg = Config::default();
+        cfg.apply_custom_layer(conf);
+        Ok(cfg)
+    }
+
+    /// Build out a config for a workspace member, which is interested in the following things,
+    /// in ascending priority:
+    /// - "root" config keys that are inherited
+    /// - axoproject stuff
+    /// - its own oranda.json
+    /// - autodetect
+    pub fn build_workspace_member(
+        config_path: &Utf8PathBuf,
+        root_config_path: &Utf8PathBuf,
+        project_root: &Utf8PathBuf,
+    ) -> Result<Config> {
+        let member_conf = OrandaLayer::load(config_path)?;
+        let root_conf = OrandaLayer::load(root_config_path)?;
+        let project = AxoprojectLayer::load(Some(project_root.into()))?;
+        let mut cfg = Config::default();
+        cfg.apply_custom_layer(root_conf);
+        cfg.apply_project_layer(project);
```

Comment:
This ordering is very interesting, did you have a case in mind where this works Better than axoproject->oranda-workspace.json->oranda.json?

Reply:
Also HMM it's unfortunate that axoproject is being invoked for every package, when it already computes info for every package in a cargo workspace unconditionally. But it's not as simple as running it only once, because we allow for flexible members that might not be properly in a cargo workspace. Feels like we want to have some kind of cache where we check if previous runs of axoproject picked up this dir, and only rerun if not...?

hmm hmm hmm I need to think about the API design for axoproject here

Reply:
hrrrrrm, i am not sure of the exact details of what you mean here, but i think in general explicit oranda config should *definitely* override all other configs, largely due to it being the only config within our and our users direct control. anything that axoproject suggests should be modifiable (with additions or substractions) by an oranda-config. it's possible this leads a user to errors, but they should be easier to fix than errors in axoproject.

Reply:
>  explicit oranda config should definitely override all other configs

it does, you can ofc still override all things in the workspace member config, my first thought was just that axoproject should have priority over what's passed down from the workspace root

Reply:
(happy to change this, ofc)